pipelines/full-build.yaml

```diff
@@ -839,3 +843,28 @@ stages:
               docker push $IMAGE --all-tags
             done
           displayName: Deploy Image
+    - job: update_odm_docs
+      displayName: "Update Assemblyline4 Documentation"
+      steps:
+        - download: current
+          artifact: working
+        - checkout: DocsRepo
+          path: $(Pipeline.Workspace)/working/assemblyline4_docs
+
+        - script: |
+            set -xv  # Echo commands before they are run
+            export TAG=${BUILD_SOURCEBRANCH#"refs/tags/v"}
+            if [[ "$TAG" == *stable* ]]
+            then
+              # Generate documentation from script
+              python gen_docs.py . "/assemblyline4_docs"
+
+              # Commit documentation changes to Documentation repo
+              cd assemblyline4_docs
+              git add .
```

Comment:
We should make sure we don't do an empty commit an check if there were any files changes.